.github/workflows/deploy.yaml

```diff
@@ -1,7 +1,13 @@
-name: CI
+name: Deploy
 
 on:
-  workflow_dispatch: {}
+  workflow_dispatch:
+    inputs:
+      gitRef:
```

Comment:
Also, do we really want to support/encourage deploying specific commits? It might be wiser for us to stick with the default (which just shows branches and tags).

Reply:
Ignore me - I'd missed that that default dropdown is about where the workflow source comes from, not the branch that the workflow operates on.

Reply:
So agree, we don't want to encourage commit deploys - but in our current infra we support it, so just keeping abilities on par. But typically developers tend deploy specifying the branch name anyways, so don't think it'll be a big issue.

Reply:
Also annoyingly, I had a look but couldn't see a way to use the nice tag/banch autocomplete input for other custom inputs on workflows. So just had to be a string based input :(